app/src/main/java/org/gdg/frisbee/android/app/OrganizerChecker.java

```diff
@@ -1,85 +1,90 @@
 package org.gdg.frisbee.android.app;
 
 import android.content.SharedPreferences;
+import android.text.format.DateUtils;
 
 import com.google.android.gms.common.api.GoogleApiClient;
 
-import org.gdg.frisbee.android.Const;
 import org.gdg.frisbee.android.api.Callback;
 import org.gdg.frisbee.android.api.model.OrganizerCheckResponse;
 import org.gdg.frisbee.android.utils.PlusUtils;
 import org.gdg.frisbee.android.utils.PrefUtils;
 
 public class OrganizerChecker {
-    private boolean mIsOrganizer = false;
-    private long mLastOrganizerCheck = 0;
-    private String mCheckedId = null;
-    private SharedPreferences mPreferences;
 
-    public OrganizerChecker(SharedPreferences preferences) {
-        mPreferences = preferences;
-        initOrganizer();
-    }
+    private static final String PREF_ORGANIZER_CHECK_TIME = "pref_organizer_check_time";
```

Comment:
pref keys resp. cache keys should be part of preferences resp. model cache to avoid clashes, they could also remain global IMHO